src/components/App.tsx

```diff
@@ -163,10 +163,10 @@ export class App extends React.Component<AppProps, AppState> {
         { min: 40, value: 256 }
       ],
       editorSplits: [],
-      toasts: [],
       showProblems: true,
       showSandbox: true
     };
+    this.toastInstance = new ToastContainer();
```

Comment:
You should not create the instance here. The ref assignment does this for you in the render() function.